src/definition/bazel_goto_definition_provider.ts

```diff
@@ -19,15 +19,48 @@ import {
   Position,
   TextDocument,
   Uri,
+  window,
```

Comment:
Both these extra imports seem to be unused.

Reply:
fixed.